fw/http2.c

```diff
@@ -484,14 +484,26 @@ tfw_h2_stream_xmit_prepare_resp(TfwStream *stream)
 	}
 
 	stream->xmit.h_len = resp->mit.acc_len;
-	stream->xmit.b_len = TFW_HTTP_RESP_CUT_BODY_SZ(resp);
-	/*
-	 * Response is chunked encoded, but it is not a response
-	 * on HEAD request.
-	 */
-	if (test_bit(TFW_HTTP_B_CHUNKED, resp->flags)
-	    && !test_bit(TFW_HTTP_B_VOID_BODY, resp->flags))
-		r = tfw_http_msg_cutoff_body_chunks(resp);
+
+	if (test_bit(TFW_HTTP_B_REQ_HEAD_TO_GET, resp->req->flags)
+	    && !TFW_STR_EMPTY(&resp->body)) {
+		/* Send only headers for HEAD method. */
+		r = ss_skb_list_chop_head_tail(&resp->msg.skb_head, 0,
```

Comment:
It seems that we delete body in `tfw_http_adjust_resp` isn't it?

Reply:
`tfw_http_adjust_resp()` deletes body only for http, here we do it http2